### ISSUE TITLE
about prediction process

### ISSUE THREAD

**Issue by JunyaoHu** [YingqingHe/LVDM#12]
when training model in prediction mode, in function `add_frames_condition`, we get a mask like `[b c=1 t=4 h w]`, and we know t=0 mask = 1, and t=1,2,3 mask = 0, just like the figure in your paper. 

![image](https://github.com/YingqingHe/LVDM/assets/67564714/6d5a4363-e19d-4b11-87ae-2d25702af10f)

so `mask.all()` should be False, and `mask.all() == 0` should be True, and `is_uncond` should be True...?

So I can't get the conditional frame in the next step...

[about function all](https://pytorch.org/docs/stable/generated/torch.all.html#torch.all)
![image](https://github.com/YingqingHe/LVDM/assets/67564714/099ff91a-a02f-495a-9de5-8751d149bff9)

https://github.com/YingqingHe/LVDM/blob/2fb64c247b1991128481e375a48f7b0be3070e40/lvdm/models/ddpm3d.py#L1341-L1373
